cmake/test.cmake

```diff
@@ -97,6 +113,13 @@ main(module=${MODULE_NAME})
 ")
     elseif(PREFIX_PYTHON_COMMAND)
         set(PYTEST_CODE "${PREFIX_PYTHON_COMMAND}")
+    elseif(PREFIX_PYTHON_SCRIPT)
+        if (PREFIX_INTERACTIVE)
+            set(MEL_EXECFILE_COMMAND "python(\"execfile(\\\"${PREFIX_PYTHON_SCRIPT}\\\")\")")
```

Comment:
This is the primary reason I opted to only support `INTERACTIVE` with `PYTHON_SCRIPT`. We quickly descend into escape sequence hell trying to make sure Python code stored in a CMake variable survives intact when filtered through the guts of CMake, through the command execution, and into Maya's `-command` option.